projects/laji/src/app/shared-modules/observation-map/observation-map/observation-map.component.ts

```diff
@@ -303,7 +284,9 @@ export class ObservationMapComponent implements OnInit, OnChanges, OnDestroy {
       const dataOptions = this.lajiMap.map.getData()[dataIdx];
       if (!dataOptions) { return; }
       // Force LajiMap to redraw the occurrence data with the updated this.getFeatureStyle(), which uses the updated this.visualizationMode.
-      this.lajiMap.map.redrawDataItem(dataIdx);
+      //this.lajiMap.map.redrawDataItem(dataIdx);
+      //this.lajiMap.map.getData()[dataIdx].groupContainer.refreshClusters();
+      this.lajiMap.setData(this.lajiMap.data); // HACK: force redraw
```

Comment:
I did some testing and didn't notice issues with the commented-out way of doing this. Maybe the issue was solved by laji-map changes, so we could revert this?